### PR TITLE
feat: add base email template

### PR DESCRIPTION
**Pull request by Harish-osmosys**
## Updated Forget Password Mail
image

## Base Template 
Screenshot 2024-02-21 134009

## Summary by CodeRabbit

- **New Features**
    - Updated the base email template for password reset emails with improved styling and structure.
    - Enhanced the "Forget Password" email template with a new layout, logo, and contact information.

### CODE REVIEW

File: DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html [hunk 1]
```diff
@@ -3,10 +3,54 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Forget Password Token</title>
+    <title>Forget Password</title>
```
[the remark this answers is not in the document]
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html]: fixed

File: DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html [hunk 2]
```diff
@@ -3,10 +3,54 @@
...
+      style="
+        width: 100%;
+        max-width: 600px;
+        margin: 20px auto;
+        border-radius: 8px;
+        overflow: hidden;
+        box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
+        background: #e5ebe1;
+        font-family: 'Arial', sans-serif;
+      "
```
[the remark this answers is not in the document]
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html]: fixed

File: DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html [hunk 3]
```diff
@@ -3,10 +3,54 @@
...
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td colspan="2" style="padding: 20px">
+            <p>Hello,</p>
+            <p>
+              <h3>Use this token to reset your password.</h3>
+              <p><strong>Token :</strong> {body}</p>
```
[the remark this answers is not in the document]
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Domain/Templates/Emails/ForgetPasswordTemplate.html]: fixed

File: DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/BaseTemplate.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```
Comment (sameer-s-b) [inline DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/BaseTemplate.html:1]: You don't need wwwroot folder. We can remove it.

File: DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html [hunk 5]
```diff
@@ -1,12 +1,78 @@
...
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Forget Password Token</title>
-</head>
-<body>
-    <h3>Use this token to reset your password.</h3>
-    <p><strong>Token :</strong> {body}</p>
-</body>
+    <title>Password Reset</title>
```
Comment (sameer-s-b) [inline DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html]: Reset password.
The filename also should be reset password.

File: DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/BaseTemplate.html (hunk 4 above)
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/BaseTemplate.html:1]: This is created by Shreyansh , i am following his structure only

File: DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html (hunk 5 above)
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html]: update file name

File: DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html [hunk 6]
```diff
@@ -1,12 +1,78 @@
...
+              <strong>Use this token to reset your password.</strong>
+              <p><strong>Token :</strong> {body}</p>
+            </div>
+            <p>Thank you,<br />FoundationX Team</p>
+          </td>
+        </tr>
+      </tbody>
+      <tfoot>
+        <tr class="footer">
+          <td>Contact us at <strong>info@example.com</strong></td>
```
[the remark this answers is not in the document]
Comment (Harish-osmosys) [reply DotnetFoundation/DotnetFoundation.Api/wwwroot/Templates/Emails/ForgetPasswordTemplate.html]: As this is boilerplate project we don't have any contact info email for now just hard coded this

**Review by sameer-s-b — APPROVED** [review]

**Pull request merged**